pkg/tests/observability_alert_test.go

```diff
@@ -89,10 +93,11 @@ var _ = Describe("Observability:", func() {
 
 	It("[P1][Sev1][Observability][Stable] Should have the alertmanager configured in rule (alert/g0)", func() {
 		By("Checking if --alertmanagers.url or --alertmanager.config or --alertmanagers.config-file is configured in rule")
-		name := MCO_CR_NAME + "-thanos-rule"
-		rule, err := hubClient.AppsV1().StatefulSets(MCO_NAMESPACE).Get(name, metav1.GetOptions{})
+		rules, err := hubClient.AppsV1().StatefulSets(MCO_NAMESPACE).List(metav1.ListOptions{
+			LabelSelector: THANOS_RULE_LABEL,
+		})
 		Expect(err).NotTo(HaveOccurred())
-		argList := rule.Spec.Template.Spec.Containers[0].Args
+		argList := (*rules).Items[0].Spec.Template.Spec.Containers[0].Args
```

Comment:
Should we check if the length of `Items` is `>= 1`?

Reply:
add `Expect(len(rules.Items)).NotTo(Equal(0))`